.github/workflows/test.yml

```diff
@@ -39,7 +39,7 @@ env:
   GIT_AUTHOR_NAME: CI User
 
 jobs:
-  pre-commit:
+  populate-pip-cache:
```

Comment:
Is it worth it to keep this vs remove the job? We don't have similar jobs anywhere else.

Reply:
We could remove it, but I think I'd like for it to stay for now to avoid adding further changes while trying to get PRs triaged and resolved.

I think we could gain some seconds by not having it, but we would also burden pypi servers ~10 times more.

Reply:
Ah lets remove it!